packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -1493,6 +1493,7 @@ class PrioritizedAction extends Action<PrioritizedIntents> {
     assert(_selectedAction != null);
     assert(_selectedIntent != null);
     _selectedAction.invoke(_selectedIntent);
+    return null;
```

Comment:
This override should really change the return type to `void` since this particular action doesn't return anything. I believe changing the return type to void in the override is still a legal override?

/cc @gspencergoog WDYT?

Reply:
Same for some of the other `Action.invoke` overrides in this PR.

Reply:
done